LocalDevDockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.7-alpine
```

Comment:
Local docker file that caches pip requirements so docker builds are faster -- ended up not using the docker container anyways (this file & requirements.txt can go away)